node/src/service.rs

```diff
@@ -571,10 +571,14 @@ fn start_consensus(
 	// NOTE: because we use Aura here explicitly, we can use `CollatorSybilResistance::Resistant`
 	// when starting the network.
 
+	// NOTE: temporary workaround taken from here: https://github.com/frequency-chain/frequency/pull/2196
```

Comment:
Please add something along the lines of:
"remove the custom tx pool when https://github.com/paritytech/polkadot-sdk/pull/4639 is included in a stable release"